warp10/src/main/java/io/warp10/script/filler/FillerNext.java

```diff
@@ -62,4 +65,31 @@ public int getPostWindow() {
   public int getPreWindow() {
     return 0;
   }
+
+  @Override
+  public WarpScriptSingleValueFillerFunction compute(GeoTimeSerie gts) throws WarpScriptException {
+
+    // fill always sort input before.
+    final int nTicks = gts.size();
+    final GeoTimeSerie original = gts;
+    final long nextTick = GTSHelper.tickAtIndex(original, 0);
```

Comment:
This line and the following will fail if the GTS is empty

Reply:
no, it will return long min value.